src/components/Reports/AdditionalSalaryRequest/SharedComponents/AdditionalSalaryRequestSection.tsx

```diff
@@ -34,7 +34,7 @@ export const AdditionalSalaryRequestSection: React.FC<
     <div>
       <Box pb={4}>
         <Stack direction="row" gap={2} alignItems="center">
-          <Typography variant="h4">{t(title)}</Typography>
+          <Typography variant="h4">{t('{{title}}', title)}</Typography>
```

Comment:
`yarn extract` searches the repo for `t($string)` and adds those strings to the translations file. So we need `t('About this Form')` to show up somewhere for it to be translatable. `t('{{title}}')` isn't enough to get the section titles in our translations file. We need to use `t` in the context where the section titles are defined.

The same applies to the subtitle. The caller needs to pass in an already-translated subtitle.